src/main/java/com/wavefront/opentracing/WavefrontSpan.java

```diff
@@ -89,6 +93,7 @@
         setTagObject(tag._1, tag._2);
       }
     }
+    this.spanLogs = new ArrayList<>();
```

Comment:
We should do this on demand within `updateSpanLogsInternal` to avoid allocating this for all spans.

Reply:
done.